mne/epochs.py

```diff
@@ -1810,6 +1811,69 @@ def save(self, fname, split_size='2GB', fmt='single', overwrite=False,
             this_epochs.event_id = self.event_id
             _save_split(this_epochs, fname, part_idx, n_parts, fmt)
 
+    def export(self, fname, fmt="auto"):
+        """Export Epochs to external formats.
+
+        Supported formats: EEGLAB (set, uses :mod:`eeglabio`)
+
+        .. warning::
+            Since we are exporting to external formats, there's no
+            guarantee that all the info will be preserved in the external
+            format. To save in native MNE format (fif) without info loss,
+            use :func:`save` instead.
+
+        Parameters
+        ----------
+        fname : str
+            Name of the export file.
+        fmt : str
+            Format of the export. Defaults to auto, which will infer
+            format from fname's extension.
+
+        Notes
+        -----
+        For EEGLAB format, channel locations are expanded to full EEGLAB
+        format. For more details see :func:`eeglabio.utils.cart_to_eeglab`.
+        """
+        if fmt == "auto":
+            fmt = op.splitext(fname)[1]
+            if fmt:
+                fmt = fmt[1:]
+            else:
+                raise ValueError("Couldn't infer format from filename "
+                                 "(no extension found)")
+
+        fmt = fmt.lower()
+
+        if fmt == "set":
+            _check_eeglabio_installed()
+            from eeglabio.epochs import export_set
+            # load data first
+            self.load_data()
+
+            # remove extra epoc and STI channels
+            drop_chs = ['epoc', 'STI 014']
+            pick_chs = [ch for ch in self.ch_names if ch not in drop_chs]
+
+            chs = self.info["chs"]
+            cart_coords = np.array([d['loc'][:3] for d in chs
+                                    if d['ch_name'] not in drop_chs])
+            if cart_coords.any():  # has coordinates
+                # (-y x z) to (x y z)
+                cart_coords[:, 0] = -cart_coords[:, 0]  # -y to y
+                # swap x (1) and y (0)
+                cart_coords[:, [0, 1]] = cart_coords[:, [1, 0]]
+            else:
+                cart_coords = None
+
+            export_set(fname, self.get_data(picks=pick_chs),
+                       self.info['sfreq'], self.events, self.tmin, self.tmax,
+                       pick_chs, self.event_id, cart_coords)
```

Comment:
I would name parameters in the function call to explicit the function signature of export_set